README.md

```diff
@@ -1,2 +1,29 @@
+[![Build Status](https://travis-ci.org/betsybeth/brightevent-frontend-app.svg?branch=develop)](https://travis-ci.org/betsybeth/brightevent-frontend-app)
+[![Coverage Status](https://coveralls.io/repos/github/betsybeth/brightevent-frontend-app/badge.svg?branch=develop)](https://coveralls.io/github/betsybeth/brightevent-frontend-app?branch=develop)
 # brightevent-frontend-app
-An application that consumes the BrightEventAPI
+An React-Redux  application for events
```

Comment:
good work on how you have arranged information on Readme. You can do a better job by adding more information about what the application does in this introduction part.